plugins/org.eclipse.elk.alg.vertiflex/src/org/eclipse/elk/alg/vertiflex/p2relative/NodeComparator.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.vertiflex.p2relative;
+
+import org.eclipse.elk.alg.vertiflex.InternalProperties;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Comparator for sorting ElkNodes according to their y positions and model order.
+ *
+ */
+class NodeComparator implements java.util.Comparator<ElkNode> {
```

Comment:
What is the difference between this class and the InverseYNodeComparator? The difference could be controlled by a simple boolean.